native/src/main/java/io/ballerina/stdlib/nats/Handler.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2024 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
``` suggestion
 * Copyright (c) 2024 WSO2 Inc. (http://www.wso2.org).
```

Reply:
Fixed in c5b69255471f5abbdfa255588b555a3c12d8d6c5